src/nupic/support/__init__.py

```diff
@@ -431,13 +431,18 @@ def makeKey(name):
     replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"/var/run/syslog"'
   elif platform.startswith('linux'):
     replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"/dev/log"'
+  elif platform.startswith('win'):
+    replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"log"'
```

Comment:
Yes. Although command line stderr redirection is tricky, needing to use ```2>&1``` to redirect it to stdout (usually alongside ```>``` file redirect, or ```>>``` append)